tests/gfoidl.Base64.Tests/Internal/Base64EncoderTests/GetDecodedLength.cs

```diff
@@ -19,6 +19,26 @@ public void EncodedLength_is_0___0()
         }
         //---------------------------------------------------------------------
         [Test]
+        public void MaxEncodedSpan_1_to_50_given___correct_decoded_len()
```

Comment:
Please do the same here for the test-methods as noted in Base64UrlEncoderTest, or I'll fix it later.